### PR TITLE
Secondary implementation improvement

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <ext-mykhaylo.sul@here.com>

### CODE REVIEW

File: src/aktualizr_secondary/update_agent_ostree.cc [hunk 1]
```diff
@@ -12,11 +12,22 @@ bool OstreeUpdateAgent::isTargetSupported(const Uptane::Target& target) const {
...
-    installed_image_info.hash = _ostreePackMan->getCurrentHash();
-    // This is the policy on a target image name in case of ostree
-    // The policy in followed and implied in meta-updater (garage-sign/push) and the backend
-    installed_image_info.name = _targetname_prefix + "-" + installed_image_info.hash;
     installed_image_info.len = 0;
+    installed_image_info.hash = _ostreePackMan->getCurrentHash();
+
+    // TODO: consider more elegant way of storing currently installed target name
+    // usage of the SQLStorage and ostree implementions aimed for Primary is
+    // a quite overhead for Secondary
```
Comment (patrickvacek) [inline src/aktualizr_secondary/update_agent_ostree.cc]: What would you propose? We can make a ticket to reconsider this, but if we have to use the same storage and OSTree manager classes, it doesn't seem like much of an extra burden here.

Comment (mike-sul) [reply src/aktualizr_secondary/update_agent_ostree.cc]: Yeah, I have a few ideas, better to talk/discuss it during a call/meeting.

**Comment by mike-sul** [comment]
> Looks good but CI ain't happy.

Yeah, I am working on it. I tested this fix on QEMU and it does work for me. Also, I am going to add a fix for the second minor issue into this PR (2) Secondary outputs an incorrect hash value just after applying the pending hash/version into logs, but it does not break anything)

**Comment by mike-sul** [comment]
@patrickvacek I verified this version of QEMU multiple times with different value of OSTREE_BRANCHNAME and different reboot sequence, looks like it works and fixes the Unu issue.

**Pull request merged**